### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by link78**
Adding FasoPool info

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -2505,5 +2505,19 @@
...
+    "ticker":"FASO",
+    "poolId":"70ab9e73071c50927631a218137d53e765665a578b744feeae3c5d81",
+    "social":{
+      "twitter":"@KAD1212",
+      "website":"https://fasostakepool.com/",
+      "reddit":"",
+      "instagram":"",
+      "youtube":""
+    },
+    "telegram":"https://t.me/fasopool",
```
Comment (leo42) [inline registry.json:2519]: Please use a personal telegram handle (no groups or channels) this for Voting.

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (link78) [reply registry.json]: I just changed it

**Review by Bez625 — APPROVED** [review]
Telegram user looks good now

File: registry.json [hunk 2]
```diff
@@ -2744,5 +2744,19 @@
...
+  },
+  {
+    "ticker":"FASO",
+    "poolId":"70ab9e73071c50927631a218137d53e765665a578b744feeae3c5d81",
+    "social":{
+      "twitter":"@KAD1212",
+      "website":"https://fasostakepool.com/",
+      "reddit":"abkade",
+      "instagram":"",
+      "youtube":""
```
Comment (Bez625) [inline registry.json:2756]: Please can you add `"facebook":"",` it's a required field

**Review by Bez625 — CHANGES_REQUESTED** [review]
Failed validation, please add missing field

**Comment by link78** [comment]
Facebook just added

File: registry.json (hunk 2 above)
Comment (link78) [reply registry.json:2757]: Facebook just been added

**Comment by link78** [comment]
Fixed!
Facebook is added

**Review by Bez625 — APPROVED** [review]

**Comment by link78** [comment]
Thanks

**Pull request merged**